### ISSUE TITLE
timeout: poor error response when attempting to upload file you can't read

### ISSUE THREAD

**Issue by monkeyiq** [filesender/filesender#233]
If you select a file that you can not read the upload does not progress and you end up with a gateway timeout. If you can not read any bytes from the file it might be nice to try to report that instead of what some folks might think is a service or network problem.

**Comment by WebSpider** [comment]
I think this is covered as an enhancement in your [bootstrap](/filesender/filesender/tree/2021/bootstrap) branch?

**Comment by monkeyiq** [comment]
Yep, it has hit me a few times in the past wondering if I broke some of the code only to see that I didn't have read permission on a test file. I might port that code over from my bootstrap branch to the development one as I find it handy and suspect there are some other who might also get hit by it. For example, uploading a log file that you don't realize that your user can not read.

**Comment by WebSpider** [comment]
I actually had this happen recently in similar software. The behaviour of that software was to prevent adding it to the queue of files to be processed. Might be something to consider here as well

**Comment by WebSpider** [comment]
@monkeyiq was this ever added to 3.0rc? And do we still want to add it?
